### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by linnikvv** [ffplayout/ffplayout#203]
Hello, my friend. I really like your solution, but at the moment I can't install on a clean Debian/Ubuntu server. Could you make an auto-installer to install your software. And I would also like to see xmltv at least by the file name. Before subsequent releases, I used an auto-installer, but this time I can’t install it because I don’t really understand linux. Thanks))

**Comment by jb-alvarado** [comment]
Hello, now the [installation](https://github.com/ffplayout/ffplayout/blob/master/docs/install.md) should be more easy then before. just follow the instruction.

**Comment by linnikvv** [comment]
In particular, I had problems installing and compiling ffmpeg, could you give more detailed instructions on setting up all the necessary software.

**Comment by jb-alvarado** [comment]
Normally you can you the system ffmpeg, as long as its version is 4.2 or higher. So you can run `apt install ffmpeg`.

**Comment by cooliobr** [comment]
@jb-alvarado can i post here uma step by step for streaming server in RAM ( nginx ) and ffmpeg with nvidia (nvenc ) support ? ffmpeg from apt is out of date.

**Comment by jb-alvarado** [comment]
@cooliobr, yes you are welcome! If you want, you can also create a new issue and we can pin it on top.
